templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -400,7 +400,7 @@
                                         }]) %}
                                     {% endfor %}
                                     <participation-phases
-                                        autoswitch-hint="{{ 'period.autoswitch.hint'|trans({ phase: 'procedure.phases.internal.evaluating'|trans }) }}"
+                                        autoswitch-hint="{{ 'period.autoswitch.hint'|trans({ phase: 'procedure.phases.internal.analysis'|trans }) }}"
```

Comment:
i don't thnk so, the old one exists not anymore, i mean this should the last step and i think evaluating is the right one. Otherwise it's already overwrriten in other projects who need other text